### DISCUSSION TITLE
GitLab Git Provider | Maximum supported version

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#440] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

v12.0.3

### Description

Hi guys,
I found on [this documentation page](https://docs.mia-platform.eu/docs/11.x/self_hosted/self_hosted_requirements) that we support GitLab, **as Git Provider**, from version 14.X.

Is there a maximum supported version?
A client is about to update to **GitLab v16.9.2**, is that ok?

Thanks.

**Comment by fredmaggiowski** [comment, marked as answer]
Hi

What Console version is being used? You specified v12.0.3 in the discussion but linked a page from the v11 docs. 

Anyhow GitLab 16 is supported.

**Reply by frengooo** [reply]
Hi @fredmaggiowski,
thanks for the reply.

Sorry, the link to the v11 doc was a mistake, the client has the v12.0.3.

Thanks.
